src/PythonMigrationViewExtension/MigrationAssistant/PythonMigrationAssistantViewModel.cs

```diff
@@ -125,5 +139,22 @@ private void SetInlineViewModel()
         }
 
         #endregion
+
+        internal void DisableMigrationAssistanWarning()
+        {
+            var filePath = GetMigrationAssistantSettingsFile();
+
+            var timeStamp = DateTime.Now.ToString();
+            var machineName = Environment.MachineName;
+
+            var file = new FileInfo(filePath);
+            file.Directory.Create();
+            File.WriteAllText(file.FullName, string.Format("{0} {1}", timeStamp, machineName));
+        }
+
+        private string GetMigrationAssistantSettingsFile()
+        {
+            return Path.Combine(warningDismissPath, dynamoVersion.ToString(), disableMigrationAssistantWarningFileName);
```

Comment:
spotted in the GIF that the warning dismiss file name used the Dynamo version including the build number (`2.8.0.xxx`). Perhaps we should only include `major` and `minor`, so it'd be `2.8`, otherwise users might get the warning back after every Dynamo update, no matter how small ? Would also match how preferences etc are handled.

What do you think @mjkkirschner @Amoursol ?

Reply:
I agree, please only keep `major` and `minor` build numbers @radumg